src/components/SoundBox/ColumnOfBoxes/ColumnOfBoxes.js

```diff
@@ -9,14 +10,25 @@ export const ColumnOfBoxesWrapper = styled.div`
   margin-left: 1px;
 `;
 
-class ColumnOfBoxes extends Component {
-  render() {
-    const { color } = this.props;
-    const simpleBoxes = [1, 2, 3, 4].map(el => (
-      <SimpleBox key={el} color={color} />
-    ));
-    return <ColumnOfBoxesWrapper>{simpleBoxes}</ColumnOfBoxesWrapper>;
-  }
-}
+export const ColumnOfBoxes = ({ clickedBox, color, toggleClickedBox }) => {
+  const simpleBoxes = [1, 2, 3, 4].map(el => (
+    <SimpleBox
+      onClick={toggleClickedBox.bind(this, el)}
+      clicked={el === clickedBox}
+      key={el}
+      color={color}
+    />
+  ));
+  return <ColumnOfBoxesWrapper>{simpleBoxes}</ColumnOfBoxesWrapper>;
+};
 
-export default ColumnOfBoxes;
+export default compose(
+  withState('clickedBox', 'toggleBox', false),
+  withHandlers({
+    toggleClickedBox: ({ toggleBox, clickedBox }) => {
```

Comment:
Let's make a reducer style here, we just need to track all box and selected box, just reduce clicked(let's rename it to selected) collection and pass here.